README.md

```diff
@@ -222,6 +222,18 @@ export default {
 };
 ```
 
+##### With [`Netlify Functions`](https://docs.netlify.com/functions/overview/)
+
+in `<functionsDirectory>/graphql.mjs`
```

Comment:
What does this mean?

Reply:
netlify has a config file like this:

```toml
[functions]
directory = "myfunctiondir"
```

Reply:
I think this should be implicit knowledge coming from Netlify's documentation.

```suggestion
```

Reply:
@enisdenjo true, but thus hasn't changed in years, and the other examples show how to serve at `graphql/` yes?

Reply:
Other examples are just code, no configuration.

Reply:
If you want, you can point to the relevant Netlify Functions documentation. But, lets not leave _some_ configuration without enough context - I think it'll just raise more confusion.

Reply:
@enisdenjo for example, here fastify is configured to serve graphql at the `/graphql` path:
https://github.com/graphql/graphql-http#with-fastify